.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
+	// Use IntelliSense to learn about possible attributes.
+	// Hover to view descriptions of existing attributes.
+	// For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
Are comments allowed in these JSON files?

Reply:
Comments are not allowed in JSON but VSCode doesn't complian about it if added. Infact here VSCode added it on its own so maybe its better to keep them as they can convery more meaning.

Reply:
For all its settings, VSCode employs `json` files described as "JSON with Comments", that allow for the use of such comments.